neuvue_project/templates/about.html

```diff
@@ -82,7 +82,7 @@ <h2 class="pt-5 pb-3"> Collaborators </h2>
         <h2 class="pt-5 pb-3"> Contact Us </h2>
 
         <p>
-            We'd love to hear from you! Reach out at <a class="text-secondary-color-activated" href="mailto:info@neuvue.io">info@neuvue.io</a> with questions or if you are interested in collaborating with us.
+            We'd love to hear from you! Reach out at <a class="text-secondary-color-activated" href="mailto:info@neuvue.io">info@bossdb.org</a> with questions or if you are interested in collaborating with us.
```

Comment:
Very glad you are changing this. Can you change the `mailto:` link as well?

Reply:
consider it done!